packages/apm/src/integrations/tracing.ts

```diff
@@ -182,6 +182,20 @@ export class Tracing implements Integration {
   public setupOnce(addGlobalEventProcessor: (callback: EventProcessor) => void, getCurrentHub: () => Hub): void {
     Tracing._getCurrentHub = getCurrentHub;
 
+    if (global.performance) {
+      // The Performance object has a limited buffer size, often 150 entries. At some point the buffer may overflow, in
+      // which case we would not be able to use it to create/update spans.
+      // https://developer.mozilla.org/en-US/docs/Web/API/Performance
+      const oldCallback = performance.onresourcetimingbufferfull;
+      performance.onresourcetimingbufferfull = function(_event: unknown): void {
+        logger.warn('[Tracing]: Resource Timing Buffer is FULL! Increasing it to 300');
+        performance.setResourceTimingBufferSize(300);
```

Comment:
This is not guaranteed to work. Also no way to query the current size, so 300 might be actually reducing the capacity!
https://developer.mozilla.org/en-US/docs/Web/API/Performance/setResourceTimingBufferSize

![image](https://user-images.githubusercontent.com/88819/76500412-cdb38400-6440-11ea-9f8d-7efda516c34a.png)

I don't know where it will end, I'm able to stuff a lot of mark entries, way more than 150~300.

Reply:
Firefox also allows for an ever growing number of entries:

![image](https://user-images.githubusercontent.com/88819/76501469-a067d580-6442-11ea-82e7-81685a185182.png)

Reply:
As @dashed noted, the resource buffer is independent, so testing with:

```js
performance.onresourcetimingbufferfull = () => console.log("FULL")
for (let i =0;i< 1000; i++){ fetch('/favicon.ico') }
performance.getEntriesByType('resource').length
```

Without changing the buffer size, FF stored up to 250 entries.

The callback fires for every new entry (multiple calls once the buffer is full).

Reply:
Chrome:

![image](https://user-images.githubusercontent.com/88819/76502517-6bf51900-6444-11ea-9763-1fa88aebe21f.png)